core/templates/services/playthrough-issues-backend-api.service.spec.ts

```diff
@@ -16,26 +16,22 @@
  * @fileoverview Unit tests for the issues backend api service.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// PlaythroughIssuesBackendApiService.ts is upgraded to Angular 8.
-import { LearnerActionObjectFactory } from
-  'domain/statistics/LearnerActionObjectFactory';
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+import { PlaythroughIssuesBackendApiService } from
+  'services/playthrough-issues-backend-api.service';
 import { PlaythroughIssueObjectFactory } from
   'domain/statistics/PlaythroughIssueObjectFactory';
-import { PlaythroughObjectFactory } from
-  'domain/statistics/PlaythroughObjectFactory';
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-
-require('services/playthrough-issues-backend-api.service.ts');
-
-describe('PlaythroughIssuesBackendApiService', function() {
-  var PlaythroughIssuesBackendApiService = null;
-  var $httpBackend = null;
-  var PlaythroughIssueObjectFactory = null;
-  var PlaythroughObjectFactory = null;
-  var CsrfService = null;
-  var backendIssues = [{
+
+describe('PlaythroughIssuesBackendApiService', () => {
+  let playthroughIssuesBackendApiService:
+    PlaythroughIssuesBackendApiService = null;
+  let httpTestingController: HttpTestingController;
+
+  let playthroughIssueObjectFactory: PlaythroughIssueObjectFactory = null;
+  let backendIssues = [{
```

Comment:
Please group logically-similar objects together, and split each group by newlines.

Here, the logically-similar objects would be the factories/services vs. the test-data.